woocommerce/woocommerce
# Wizard: Add Stripe Email

This PR adds an email input field to the payments wizard step at `wp-admin/?page=wc-setup&step=payment`. The mockup:

<img width="443" alt="screen shot 2017-09-30 at 9 48 49 pm" src="https://user-images.githubusercontent.com/11487924/31103492-da7e7c82-a7df-11e7-8e80-922f8a16f188.png">

**To test**
1. Check this branch out
2. Go to `wp-admin/?page=wc-setup&step=payment`
3. Make sure that it works

cc @olaolusoga  @jeffstieler
